worker/src/Settings.cpp

```diff
@@ -10,9 +10,13 @@
 #include <cerrno>
 #include <iterator> // std::ostream_iterator
 #include <sstream>  // std::ostringstream
+#ifdef _MSC_VER
+#include "opt/getopt.h"
+#else
 extern "C" {
 #include <getopt.h>
 }
+#endif // _MSC_VER
```

Comment:
So we can remove the if else here, since it would always be `#include <getopt.h>`:

* In Linux/Mac it will take the system header since `getopt` is not included in gyp "dependencies".
* In Windows it will take the `getopt` in deps/ folder since `getopt` is included in gyp "dependencies".

Reply:
already changed.